### ISSUE TITLE
AniDex Receiving 404 Error

### ISSUE THREAD

**Issue by AterArdeo** [Jackett/Jackett#1720]
Jackett version: 0.7.1644.0
Mono version:  5.2.0.215

-------------------------------

Anidex stopped working recently and upon checking the logs to see why, I noticed it's getting a 404 response when searching for any items:

```
8/24/2017 9:28:18 PMSystem.Web.Http.Action: ReflectedHttpActionDescriptor ExecuteAsync Invoking action 'Test()'
8/24/2017 9:28:18 PM08-24 22:28:18 Debug IWebClient.GetString(Url:https://anidex.info/ajax/page.ajax.php?page=torrents&filename=&category=&lang_id=)
8/24/2017 9:28:19 PM08-24 22:28:19 Debug IWebClient: Returning NotFound => <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
8/24/2017 9:28:19 PM<html><head>
8/24/2017 9:28:19 PM<title>404 Not Found</title>
8/24/2017 9:28:19 PM</head><body>
8/24/2017 9:28:19 PM<h1>Not Found</h1>
8/24/2017 9:28:19 PM<p>The requested URL /ajax/page.ajax.php was not found on this server.</p>
8/24/2017 9:28:19 PM<hr>
8/24/2017 9:28:19 PM<address>Apache/2.4.18 (Ubuntu) Server at anidex.info Port 443</address>
8/24/2017 9:28:19 PM</body></html>
8/24/2017 9:28:19 PM
8/24/2017 9:28:19 PM08-24 22:28:19 Info Found 0 releases from Anidex
```

It results in the error
```
8/24/2017 9:28:19 PM08-24 22:28:19 Error Exception in test_indexer System.Exception: Found no results while trying to browse this tracker
8/24/2017 9:28:19 PM  at Jackett.Services.IndexerManagerService+<TestIndexer>c__async0.MoveNext () [0x000ef] in <3c02d63f4dc54c69812f99444f189d7e>:0 
8/24/2017 9:28:19 PM--- End of stack trace from previous location where exception was thrown ---
8/24/2017 9:28:19 PM  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <204f770036d441bb8dfd3daba3550e83>:0 
8/24/2017 9:28:19 PM  at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) [0x0003e] in <204f770036d441bb8dfd3daba3550e83>:0 
8/24/2017 9:28:19 PM  at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) [0x00028] in <204f770036d441bb8dfd3daba3550e83>:0 
8/24/2017 9:28:19 PM  at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) [0x00008] in <204f770036d441bb8dfd3daba3550e83>:0 
8/24/2017 9:28:19 PM  at System.Runtime.CompilerServices.TaskAwaiter.GetResult () [0x00000] in <204f770036d441bb8dfd3daba3550e83>:0 
8/24/2017 9:28:19 PM  at Jackett.Controllers.AdminController+<Test>c__async5.MoveNext () [0x0012b] in <3c02d63f4dc54c69812f99444f189d7e>:0 
8/24/2017 9:28:19 PM08-24 22:28:19 Info ReflectedHttpActionDescriptor ExecuteAsync Action returned 'System.Web.Http.Results.JsonResult`1[Newtonsoft.Json.Linq.JToken]' 
8/24/2017 9:28:19 PMSystem.Web.Http.Action: ReflectedHttpActionDescriptor ExecuteAsync Action returned 'System.Web.Http.Results.JsonResult`1[Newtonsoft.Json.Linq.JToken]'
```

I tried finding a similar issue, but the closest I could find was [AniDex Test being broken](https://github.com/Jackett/Jackett/issues/1444) by the category ID, which was supposedly fixed.

My log for the event is here: 
[log.txt](https://github.com/Jackett/Jackett/files/1250857/log.txt)

**Comment by GitHub010931** [comment]
having same issue, anidex wont load

**Comment by kaso17** [comment]
fixed via 92bda2b09b872a431a37b8a0cd1fb29ea48710a4
